common/components/componentsBySection/CreateEvent/AboutEventDisplay.jsx

```diff
@@ -118,6 +119,7 @@ class AboutEventDisplay extends React.PureComponent<Props, State> {
         {!_.isEmpty(event.event_legacy_organization) && (
           <ProfileProjectSearch viewOnly={this.props.viewOnly} />
         )}
+        <MainFooter key="main_footer" forceShow={event.show_headers}/>
```

Comment:
The MainFooter component was designed to be full-width, and inside the "AboutEvent-root container" div puts it within a smaller frame.  I would either put it in the AboutEventController component, or just outside the div in this component.

Reply:
Done and tested.